composer.json

```diff
@@ -6,21 +6,21 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/core": "^8.7",
-        "php": ">=7.2",
+        "drupal/core": "^8.9 || ^9",
+        "php": ">=7.3",
         "nesbot/carbon": "^2.29"
     },
     "require-dev": {
         "composer/installers": "~1.5",
-        "drupal/core-composer-scaffold": "^8.8",
-        "drupal/config_devel": "~1.2",
+        "drupal/core-composer-scaffold": "^8.9 || ^9",
+        "drupal/config_devel": "^1.8",
         "drupal/drupal-extension": "~4.0",
-        "drush/drush": "~9.0@stable",
+        "drush/drush": "~10.3",
         "guzzlehttp/guzzle": "~6.3",
```

Comment:
```suggestion
```
We don't need `"guzzlehttp/guzzle": "~6.3",` because it is already the dependency of the Drupal core